lua-libraries/lunamark/reader/markdown.lua

```diff
@@ -890,11 +891,14 @@ function M.new(writer, options)
   -- Basic parsers (local)
   ------------------------------------------------------------------------------
 
+  local specials = "*_~`&[]<!\\-@^"
```

Comment:
(Aside node) If going this route, we should perhaps fix the changes made for superscripts and subscripts the same way, to avoid having the added characters being interpreted as "specials" when the corresponding extensions are disabled.